docs/tab-autocomplete.md

```diff
@@ -31,8 +31,22 @@ PearAI supports tab autocomplete, which predicts and suggests what you would typ
 
 3. **Enjoy the development speed up with autocomplete!**
 
-## Alternative
-
-- You can also use [Supermaven](https://supermaven.com/) for tab autocomplete. It is currently one of the best autocomplete AI on the market, and provides a free tier. You can get started by installing Supermaven directly as an extension within PearAI.
+## Alternatives
 
+- You can use [Supermaven](https://supermaven.com/) for tab autocomplete. It is currently one of the best autocomplete AI on the market, and provides a free tier. You can get started by installing Supermaven directly as an extension within PearAI.
+  
   ![Supermaven extension](../static/img/supermaven.png)
+
+- Also, you can use [Ollama](https://ollama.ai/) local models.  Download the model that you want and add it to your config.json file. Examples:
+
+  ```json
+  "tabAutocompleteModel": {
+      "title": "Codegemma",
```

Comment:
how are these local models in your experience if you have tried them? nang had tried ollama and said it was ass XD

Reply:
Honestly, Ollama local models depend on how much vram your computer has. If you aren't running a computer with a good GPU, they are gonna be slow and I think it does affect the quality of the code completions. 

I have had good quality code completions with StarCoder2 and Deep-seek. However, I couldn't run Deep-seek 2, which I heard is even better than the original. 

I do know from some discussion boards that many devs are using local models for tab auto completions, so I wanted to add this to the documentation as another option, and it lets users know that it is an option, and that PearAI does indeed support local models. 

Oh, but I had crappy luck with Codegemma. It sucked. So, I will update my PR and remove it. I am not sure why I through that one in there.

Reply:
@Fryingpannn PR has been updated.  I left StarCoder2 as an example Ollama model that a user could configure.  Really, Codestral has been what I have been using, and it is great!  I recommend it over our other choices.  But it is hard to find a good model that is tab autocomplete specific.  I have found it is hard to find reliable reviews.